Transformers/HinglishBERTFinetuning.ipynb

```diff
@@ -0,0 +1,978 @@
+{
```

Comment:
<p>Why use e-7???</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Welll--It worked better. </p>